rust/agama-lib/src/users/store.rs

```diff
@@ -40,12 +40,22 @@ impl UsersStore {
         })
     }
 
+    // small helper to convert dbus empty string to None
+    fn optional_string(value: String) -> Option<String> {
```

Comment:
I would put this function in https://github.com/agama-project/agama/blob/master/rust/agama-lib/src/dbus.rs